### PR TITLE
Community CHSH game update

### PR DESCRIPTION
**Pull request by muneerqu**
Update CHSH game to run on qiskit 0.6

### CODE REVIEW

**Comment by rraymondhp** [comment]
Thanks @muneerqu 
Can you take a look at the algorithm as well? I suspect there is a bug in the notebook because the quantum strategy should produce better winning probabilities than the best classical strategy. But the current notebook produces otherwise.

**Comment by muneerqu** [comment]
Hi @rraymondhp thank you for reviewing the notebook. In the updated version I didn't choose the best winning strategy for both classical (my choice was 2,3) and quantum (my choice was 2,3) systems.

When I choose the best winning strategy for classical (1, 4) and quantum (3, 3) I get around 75% and 85% winning probability respectively for classical and quantum systems.

Please let me know if I need to review it again. Also, I can update the notebook to demonstrate the winning strategy and the advantage of the quantum system.

**Comment by muneerqu** [comment]
I agree. I will make a subsection and explain that.

**Comment by muneerqu** [comment]
I added a section at the end to show the advantage of the quantum strategy over the classical one.

File: community/teach_me_qiskit_2018/chsh_game/CHSH game-tutorial.ipynb [hunk 1]
```diff
@@ -518,6 +526,94 @@
...
+    "$$\n",
+    "P_{0,0} + P_{0,1} + P_{1,0}+P_{1,1}\n",
+    "$$\n",
+    "\n",
+    "$$\n",
+    "cos^{2}(\\frac{\\pi}{8}) + cos^{2}(\\frac{\\pi}{8})+cos^{2}(\\frac{\\pi}{8})+sin^{2}(\\frac{3\\pi}{8})\n",
+    "$$\n",
+    "\n",
+    "$$\n",
+    "\\frac{3}{4} cos^{2}(\\frac{\\pi}{8}) + \\frac{1}{4} sin^{2}(\\frac{3\\pi}{8}) = \\approx 85\\%\n",
```
Comment (rraymondhp) [inline community/teach_me_qiskit_2018/chsh_game/CHSH game-tutorial.ipynb]: Please replace `= \approx` with just `\approx`.

**Comment by rraymondhp** [comment]
Thanks for the quick update. I have small requests of changes, but overall it is very good...

**Comment by muneerqu** [comment]
I think it is 85.53% some round it to 86%!

Should I change it?

**Comment by rraymondhp** [comment]
I think it is 1/2 + 1/(2*sqrt(2)) = 0.8535533905932737 which should be about 85% when rounded.

**Comment by muneerqu** [comment]
That's correct. I will change them to be 85%.

**Pull request merged**